### PR TITLE
Fix/quickorder inactive status message

### PR DESCRIPTION
**Pull request by Josmar-jr**
#### What does this PR do? \*
Add new message in the status column of the SKU table, specifically inactive skus

#### How to test it? \*
1. Enter the URL https://giubernal--b2bstore005.myvtex.com
2. Open the menu -> click at 'Quick Order'
3. Insert skus code at `Copy/Paste Skus`

#### Describe alternatives you've considered, if any. \*

https://github.com/user-attachments/assets/ab3eac32-bb44-45ac-ab35-eb71d2eaa311

#### Related to / Depends on \*
Branch created from the back task

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "editor.formatOnSave": true,
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "explicit"
```
Comment (ataideverton) [inline .vscode/settings.json:4]: Why did this change

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
```
Comment (ataideverton) [inline CHANGELOG.md:10]: Please group both entries in a single ### Fixed topic

```suggestion
### Fixed

- Fixed the message in the status column of the SKU table, specifically inactive skus
- Fixed the return from skuFromRefIds and improve the return from getSkuSellerInfo
```

File: messages/es.json [hunk 3]
```diff
@@ -35,6 +35,7 @@
   "store/quickorder.available": "Disponible",
   "store/quickorder.back": "Atrás",
   "store/quickorder.cannotBeDelivered": "No se puede entregar",
+  "store/quickorder.inactive": "No disponible",
```
Comment (ataideverton) [inline messages/es.json:38]:
```suggestion
  "store/quickorder.inactive": "Indisponible",
```

File: .vscode/settings.json (hunk 1 above)
Comment (Josmar-jr) [reply .vscode/settings.json:4]: probably when the file was saved, my vscode preferences changed this

**Pull request merged**